perspective_reddit_bot/config.yaml

```diff
@@ -7,6 +7,10 @@
 #   perspective_score: A list of models and the thresholds at which to apply the
 #                      rule. If multiple models are listed, then the action only
 #                      occurs if all thresholds are met
+#   comment_features: A list of features and values. The features are:
+#       toplevel_only: boolean feature. If true, rule only applies to top-level
```

Comment:
Should we include an example of this to the default config file? Maybe just add to one of the rules already in there?

Reply:
done